eras/shelley/impl/src/Cardano/Ledger/Shelley/API/ByronTranslation.hs

```diff
@@ -156,15 +156,15 @@ translateToShelleyLedgerState genesisShelley epochNo cvs =
       LedgerState
         { lsUTxOState =
             UTxOState
-              { _utxo = utxoShelley,
-                _deposited = Coin 0,
-                _fees = Coin 0,
-                _ppups = def,
-                _stakeDistro = IStake mempty Map.empty
+              { utxosUtxo = utxoShelley,
+                utxosDeposited = Coin 0,
+                utxosFees = Coin 0,
+                utxosPpups = def,
+                utxosStateDistro = IStake mempty Map.empty
```

Comment:
Maybe `utxoStateDistro` (thus dropping the word "stake", so "utxo state distribution"), or `utxoStStakeDistro` would be clearer? :thinking:

Reply:
I think @aniketd you mixed up `State` vs `Stake`. So it should be `_stakeDistro` -> `utxosStakeDistro`
```suggestion
                utxosStakeDistro = IStake mempty Map.empty
```

`utxos` seems like a good enough prefix for `UTxOState`